src/components/molecules/Visualizer/hooks.ts

```diff
@@ -103,6 +109,50 @@ export default ({
   );
   dropRef(wrapperRef);
 
+  const viewportMobileMaxWidth = 768;
```

Comment:
This can be put outside of the hooks since it is always the same

Reply:
Updated. Please help check if i do it correctly.